.github/workflows/stale-bot.yml

```diff
@@ -0,0 +1,33 @@
+# Go to https://github.com/actions/stale for the full documentation of these options.
+
+name: 'Stale bot'
+on:
+  schedule:
+    - cron: '30 1 * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v3.0.18
+        with:
+          # Rate limit per run, defaults to 30.
+          operations-per-run: 30
+
+          # Only issues with both `reproduction:needed` and `type:bug` will be touched by the stale bot.
+          only-issue-labels: 'reproduction:needed,type:bug'
+
+          # A issue that has any assignee is getting worked on, so is exempt from stale bot.
+          exempt-all-issue-assignees: true
+
+          # Remove stale label from issue/PR on updates or comments, defaults to true.
+          remove-stale-when-updated: true
+
+          # Wait 14 days until making issue stale and posting a message.
+          days-before-issue-stale: 14
+          stale-issue-message: 'This issue will be closed after 7 days if a reproduction is not provided.'
+          stale-issue-label: 'stale'
+
+          # We can set a grace period for contributors to provide the reproduction, and then finaly close it.
+          days-before-issue-close: 7
+          close-issue-message: 'This issue is closed because it has no reproduction.'
```

Comment:
The stale bot can also give a message, but we already have the label bot giving a message also.
We will need to decide on if we want to give a short "grace period" before closing after 14 days, or if we just want to go: "14 days have passed so we're closing now".

Reply:
One warning and then the closing is enough